configurations/s02_pipeline_configuration.py

```diff
@@ -179,7 +179,8 @@
         membership_group_configuration=MembershipGroupConfiguration(
             membership_group_csv_urls={
                 "listening_group": [
-                    "gs://avf-project-datasets/2021/WUSC-LEAP/leap_s02_listening_group.csv"
+                    "gs://avf-project-datasets/2021/WUSC-LEAP/leap_s02_listening_group.csv",
+                    "gs://avf-project-datasets/2021/WUSC-LEAP/wusc_leap_kalobeyei_s02e02_listening_group.csv"
```

Comment:
Should these be named consistently? currently it's leap_s02 for one and wusc_leap_kalobeyei_s02 for the other.

Could you also add a comment above each entry that briefly describes what it contains? As someone who doesn't generally support projects that have been using listening groups, I always get a little bit confused by who is in each of these listening group files

Reply:
We've discussed this offline and we've agreed that we'll be updating `leap_s02_listening_group.csv` with every episode's listening group. yes, I'll add a comment above the entry in another PR.